pupil-spa/src/app/auditEntry.ts

```diff
@@ -1,9 +1,33 @@
-export type AuditEntryType = 'CheckStarted' | 'QuestionRendered' | 'QuestionAnswered' | 'PauseRendered' | 'UserInput';
```

Comment:
@GeorgeChatzigiannis it is getting cluttered, but did this to avoid multiple import statements.